src/assets/locales/de/translations.json

```diff
@@ -1,8 +1,8 @@
 {
-  "main.description": "Wir sind pragmatische Profis, die zuverlässige Software für das Web entwickeln.",
-  "main.services.title": "Plattformen, Beratung & Produkte",
+  "main.description": "Wir machen die digitale Welt einfacher. Mit maßgeschneiderter Web-Software für Unternehmen. Von Frontend bis Backend. Vom Webauftritt bis zur digitalen Sicherheitsinfrastruktur.",
+  "main.services.title": "Wir verbinden. Mit Technologie.",
   "main.services.kicker": "Unsere Leistungen",
-  "main.services.text": "Wir haben großen Spaß an Technologie und freuen uns auf neue Herausforderungen. Dabei fokussieren wir uns auf langfristige Engagements im Konzerngeschäft. Neuen Aufgaben begegnen wir immer mit angemessenem Respekt. Wir streben stets hochwertige und zeitgemäße Lösungen an – die Wahl der Technologie ist für uns dabei sekundär. Gern unterstützen wir auch Sie in den folgenden Disziplinen:",
+  "main.services.text": "Unsere Software Ingenieure und Designer vereint eines: die Liebe zum Detail und zur Technologie. \u2028Beide bringen wir für die Ziele Ihres Unternehmens und die Experience Ihrer Kunden ein. So finden wir innovative und sichere Lösungen. Mit Respekt, Neugierde, Kreativität. Und jeder Menge Herzblut.",
```

Comment:
"\u2028" is not being displayed properly on my machine:
![image](https://user-images.githubusercontent.com/7970458/146550496-6be2f516-5ed9-49ac-bb56-a7dcb89843ef.png)

What was that supposed to be? A line break?

I also suggest to combine the last two sentences, like so:
```suggestion
  "main.services.text": "Unsere Software Ingenieure und Designer vereint eines: die Liebe zum Detail und zur Technologie. \u2028Beide bringen wir für die Ziele Ihres Unternehmens und die Experience Ihrer Kunden ein. So finden wir innovative und sichere Lösungen. Mit Respekt, Neugierde, Kreativität und jeder Menge Herzblut.",
```